sdk.docker.cmake

```diff
@@ -0,0 +1,27 @@
+# Cmake toolchain description file for the wasi-sdk docker image
+
+# This is arbitrary, AFAIK, for now.
+cmake_minimum_required(VERSION 3.4.0)
+
+# To make sure it recognizes the WASI platform
+list(APPEND CMAKE_MODULE_PATH /usr/share/cmake/Modules)
+
+set(CMAKE_SYSTEM_NAME WASI)
+set(CMAKE_SYSTEM_VERSION 1)
+set(CMAKE_SYSTEM_PROCESSOR wasm32)
+set(triple wasm32-wasi)
+
+set(CMAKE_C_COMPILER /usr/bin/clang-15)
+set(CMAKE_CXX_COMPILER /usr/bin/clang++-15)
+set(CMAKE_AR /usr/bin/llvm-ar-15)
+set(CMAKE_RANLIB /usr/bin/llvm-ranlib-15)
```

Comment:
Given that you set CC/CXX etc in the environment maybe this is redundant here?

Reply:
With cmake, CC/CXX do get sourced from env variables, but AR RANLIB still don't unfortunately. It makes the state clearest I think to have it all in one place here rather than relying on environment for some but not all